src/components/dataTile/DataTile.tsx

```diff
@@ -0,0 +1,18 @@
+import './DataTile.scss';
+
+type Props = {
+  children: string;
+  title?: string;
+  theme: 'blue' | 'green' | 'teal' | 'yellow';
+};
+
+const DataTile = ({ children, title, theme }: Props) => {
```

Comment:
możemy to skrócić do wersji DataTile = ({...}) => ( <div>...</div> )

Reply:
👍